Project.toml

```
@@ -1,6 +1,6 @@
 name = "OrderedCollections"
 uuid = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
-version = "1.6.3"
+version = "1.6.4"
```

```suggestion
version = "1.6.4-DEV"
```
let's maybe wait with the final version until a release is made